client/src/components/auth/ForgotPassword.vue

```diff
@@ -0,0 +1,172 @@
+                        >{{ lang.SEND_RESET_INSTRUCTION }}
+                    </VBtn>
+                    <RouterLink
+                        :to="{ name: 'SignIn' }"
+                        class="remembered-password "
+                    >
+                        {{ lang.REMEMBERED_PASSWORD }}
+                    </RouterLink>
+                </VRow>
+            </VCol>
+            <VSpacer class="pa-4" />
+            <VCol cols="11" sm="11" md="8" class="pa-0">
+                <VAlert
+                    :type="forgotPasswordData.typeResultSubmitPassword"
+                    dense
+                    outlined
+                    text
+                    dismissible
+                    :value="forgotPasswordData.helperVisibility"
+                >
+                    <h6>{{ forgotPasswordData.resultSubmitPassword }}</h6>
+                    <p>{{ forgotPasswordData.explanation }}</p>
+                </VAlert>
+            </VCol>
+        </VForm>
+    </div>
+</template>
+
+<script>
+import * as actions from '@/store/modules/auth/types/actions';
+import * as mutations from '@/store/modules/auth/types/mutations';
+import { mapActions, mapMutations, mapState } from 'vuex';
+import enLang from '@/store/modules/i18n/en';
+import { validationMixin } from 'vuelidate';
+import { required, email } from 'vuelidate/lib/validators';
+import * as notificationActions from '@/store/modules/notification/types/actions';
+
+export default {
+    name: 'ForgotPassword',
+    mixins: [validationMixin],
+    components: {},
+    data: () => ({
+        lang: enLang
+    }),
+    validations: {
+        forgotPasswordData: {
+            email: { required, email }
+        }
+    },
+    methods: {
+        ...mapMutations('auth', {
+            changeHelperVisibilityForgot:
+                mutations.CHANGE_HELPER_VISIBILITY_FORGOT,
+            setEmailForgot: mutations.SET_EMAIL_FORGOT
+        }),
+        ...mapActions('auth', {
+            forgotPassword: actions.FORGOT_PASSWORD
+        }),
+        ...mapActions('notification', {
+            setErrorNotification: notificationActions.SET_ERROR_NOTIFICATION
+        }),
+        setEmail(e) {
+            this.setEmailForgot(e.target.value);
+            this.$v.forgotPasswordData['email'].$touch();
+        },
+        setEmailOnInput(e) {
+            this.setEmailForgot(e);
+            this.$v.forgotPasswordData['email'].$touch();
+            this.changeHelperVisibilityForgot(false);
+        },
+        async onSubmit() {
+            try {
+                this.$v.$touch();
+                if (this.$v.$invalid) {
+                    throw new Error(this.lang.PLEASE_ENTER_CORRECT_DATA);
+                }
+                await this.forgotPassword();
+            } catch (error) {
+                this.setErrorNotification(error?.message);
+            }
+        }
+    },
+    computed: {
+        ...mapState('auth', ['forgotPasswordData']),
```

Comment:
probably, getter is better here?

Reply:
Sorry I do not understand where exactly  getter is better